chainer/functions/connection/convolution_2d.py

```diff
@@ -183,10 +184,26 @@ def forward_gpu(self, inputs):
                     handle, x_desc.value, filter_desc.value,
                     conv_desc.value, y_desc.value, _fwd_pref, workspace_size)
 
-            if use_tensor_core:
-                # Only CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_PRECOMP_GEMM
-                # supports Tensor-Core in cuDNN7.
-                algo = libcudnn.CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_PRECOMP_GEMM  # NOQA
+            _algos = [
+                libcudnn.CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_PRECOMP_GEMM,
+                libcudnn.CUDNN_CONVOLUTION_FWD_ALGO_WINOGRAD_NONFUSED, ]
```

Comment:
`CUDNN_CONVOLUTION_FWD_ALGO_WINOGRAD_NONFUSED` also supports Tensor-core now?
It is not clear that what `algo not in _algos` below means.
Could you assign a local variable with a clear name for both `_algos` and `algo not in _algos`.